Update temporal-table-considerations-and-limitations.md (#9735)
MicrosoftDocs/sql-docs: docs/relational-databases/tables/temporal-table-considerations-and-limitations.md

- If current table is partitioned, the history table is created on default file group because partitioning configuration isn't replicated automatically from the current table to the history table.

> - Temporal and history tables can't be FileTable or FILESTREAM, since FileTable and FILESTREAM allow data manipulation outside of [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] and thus system versioning can't be guaranteed.

Reviewer: ```suggestion
- Temporal and history tables can't use FileTable or FILESTREAM, since FileTable and FILESTREAM allow data manipulation outside of [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] and thus system versioning can't be guaranteed.
```

> - Temporal and history tables can't be FileTable and can only contain columns of any supported data type other than FILESTREAM, since FileTable and FILESTREAM allow data manipulation outside of [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] and thus system versioning can't be guaranteed.

Reviewer: ```suggestion
- Temporal and history tables can't be FileTable or FILESTREAM, since FileTable and FILESTREAM allow data manipulation outside of [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] and thus system versioning can't be guaranteed.
```
